Resources/Prototypes/_Harmony/Entities/Mobs/Player/familiars.yml

```
@@ -0,0 +1,64 @@
+﻿- type: entity
+  name: Gizard
+  parent: SimpleMobBase
+  id: MobLizardGizard
+  description: It's Gizard the Wizard Lizard! Here to help with wizarding!
+  components:
+  - type: Sprite
+    drawdepth: Mobs
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: gizard
+      sprite: _Harmony/Mobs/Pets/gizard.rsi
+  - type: GhostRole
+    makeSentient: true
+    allowMovement: true
+    allowSpeech: true
+    name: ghost-role-information-gizard-name
+    description: ghost-role-information-gizard-description
+    rules: ghost-role-information-familiar-rules
+    mindRoles:
+    - MindRoleGhostRoleFamiliar
```

This ghost role should have the same raffle settings as the spawner
```suggestion
  - type: GhostRole
    makeSentient: true
    allowMovement: true
    allowSpeech: true
    name: ghost-role-information-gizard-name
    description: ghost-role-information-gizard-description
    rules: ghost-role-information-familiar-rules
    mindRoles:
    - MindRoleGhostRoleFamiliar
    raffle:
      settings: short
```